ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -172,7 +172,7 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
     /// The default leading padding in the cell.
     @objc public static let defaultPaddingLeading: CGFloat = {
         let tokenSet = TableViewCellTokenSet(customViewSize: { .default })
-        return tokenSet[.paddingLeading].float
+        return TableViewCellTokenSet.paddingLeading
```

Comment:
If you're referencing the TableViewTokenSet this way now, do you need the above let? Also what is the rationale for this change?

Reply:
good catch! Also we are moving away from spacing/padding as customizable tokens, hence the change to constant tokens